src/ui/MLFacet/MLFacetValues/MLFacetValueRenderer.ts

```diff
@@ -60,9 +62,18 @@ export class MLFacetValueRenderer {
     this.facet.toggleSelectValue(this.facetValue.value);
     this.toggleSelectedClass();
     this.facet.enableFreezeCurrentValuesFlag();
-    this.facet.triggerNewQuery();
+    this.facet.triggerNewQuery(() => this.sendSelectionUAEvent());
   };
 
+  private sendSelectionUAEvent() {
```

Comment:
One thing to think about is the use of acronyms in names versus the conventions in the project. Do we normally use `UA`, `Analytics`, or something else in the method names? Do we use `send` or `log`? t's a little nicer when the naming is consistent.